docs/oev-searchers/in-depth/oev-auctioneer.test.ts

```diff
@@ -0,0 +1,43 @@
+import path from 'node:path';
+import fs from 'node:fs';
+import { test, expect } from 'vitest';
+import { CHAINS } from '@api3/contracts';
+import { getChains } from '@api3/dapi-management';
+
+const supportedMainnets = getChains()
+  .filter((chain) => chain.stage === 'active')
+  .map((chain) => CHAINS.find((api3Chain) => api3Chain.id === chain.id)!)
+  .filter((chain) => !chain.testnet)
+  .filter((chain) => chain.alias !== 'oev-network')
+  .map((chain) => chain.name)
+  .sort((a, b) => a.localeCompare(b));
+
+test('the auction length section lists all the supported chains', () => {
+  const docPath = path.join(
+    process.cwd(),
+    'docs',
+    'oev-searchers',
+    'in-depth',
+    'oev-auctioneer.md'
+  );
+  const content = fs.readFileSync(docPath, 'utf8');
```

Comment:
Can you make sure this errors out nicely in case the path doesn't exist (when we move the files arouind)